lib/plugins/aws/deployFunction/index.js

```diff
@@ -33,8 +33,10 @@ class AwsDeployFunction {
       'deploy:function:deploy': () => BbPromise.bind(this)
         .then(() => {
           if (!this.options['update-config']) {
-            this.deployFunction();
+            return this.deployFunction();
           }
+
+          return BbPromise.resolve();
```

Comment:
There was an error:
```
return new BbPromise().resolve();
```
Should be:
```
return BbPromise.resolve();
```

Reply:
Ah. Nice catch! Thanks @RafalWilinski 👍

Yes, I changed it because we're using `Bluebird` as our Promise solution (and we should use `Bluebird` everywhere since we had some nasty bugs related to `Promise` vs. `BbPromise` in our `Variables.js` refactoring. We basically faced race conditions since `Bluebird` was faster than `Promise`).